app/models/reservation.rb

```diff
@@ -0,0 +1,6 @@
+class Reservation < ApplicationRecord
+  belongs_to :screening
+
+  validates :quantity, presence: true, numericality: { greater_than_or_equal_to: 1 }
+  validates :status, inclusion: { in: %w[pending confirmed cancelled] }
```

Comment:
That is okay, especially that status is string.
The other solution I know is to store status as integer. And map this specific integer to [enum](https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails.html)
Benefits I can see in the enum solutions are:
- you can easilly change the mapped name of the status. Database stores integers so that want change and rails will automatically mapped them to declared enum values,
- also you have a lot of ready active record functions which you can use on enum values by default - [here](https://api.rubyonrails.org/v6.0.5/classes/ActiveRecord/Enum.html) you have example in the first code block

If you have any question about that do not hesitate let me know on slack or wherever you want.